aspnetcore/signalr/hubs.md

```diff
@@ -38,6 +38,7 @@ Create a hub by declaring a class that inherits from <xref:Microsoft.AspNetCore.
 > Hubs are [transient](/dotnet/core/extensions/dependency-injection#transient):
 >
 > * Don't store state in a property of the hub class. Each hub method call is executed on a new hub instance.
+> * Don't instantiate a hub directly via dependency injection. Hubs aren't intended to be used directly. Only interact with a hub via an [`IHubContext`](xref:signalr/hubcontext).
```

Comment:
```suggestion
> * Don't instantiate a hub directly via dependency injection.
```

I think this is what @guardrex  suggested. Can you approve that or make another suggestion?

Reply:
I think the statement to suggest what to use instead of instantiating a Hub via dpeendency injection has value, so I reworded it an pushed another commit.